packages/sitecore-jss-angular/src/components/link.directive.ts

```diff
@@ -67,7 +71,11 @@ export class LinkDirective implements OnChanges {
       ...this.getElementAttrs(),
       ...this.attrs,
     };
-    Object.keys(attrs).forEach((key) => this.renderer.setAttribute(span, key, attrs[key]));
+    Object.keys(attrs).forEach((key) => {
```

Comment:
could also use Object.entries, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

tslib is set on es2017 so entries should be available , https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-angular/tsconfig.json

Reply:
Thanks for the suggestion. I consider this change out of scope for the current PR, I will raise an issue for this.